.github/workflows/ci.yml

```diff
@@ -25,13 +25,4 @@ jobs:
           node-version: '10'
       - run: npm ci
       - run: npm run front-end:lint
-      - run: npm run back-end:lint
-  audit:
```

Comment:
This being removed temporarily as it is currently failing correct? Will put it in and fix in a separate PR at some point in the future?

Reply:
Yes, once this PR is merged into main, I'll open a new PR to add the `npm audit` back in. It can stay as a draft until the audit errors are sorted out.

Reply:
Split to #12